### PR TITLE
MilosParipovic.OneCommander version 3.44.1

### LINKED ISSUE #106353: [Package Request]: OneCommander
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://onecommander.com/
Publisher: One Commander
Package Name: OneCommander
Description: OneCommander is a modern file manager for Windows 10 and Windows 11. Features include tabs, dual-pane browsers, columns navigation, built-in preview, a theming system, color tags, and much more. It is free for home use and has no ads.
Package Version: 3.44.1.0
Installer URL: https://onecommander.com/OneCommanderSetup3.44.1.0.msi

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves #106353

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/107523)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
I called out Windows Features. This one is most likely using the app features.

**Comment by BrandonWanHuanSheng** [comment]
It look like everything install correctly

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie  What is this error causing on it.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
`user` scope for MSI installers generally does not work well because of a bug/design issue? with WiX installers in that the installer installs per-user but incorrectly writes the uninstall entries to HKEY_LOCAL_MACHINE that makes WinGet think it's a per-machine install. This causes users not being able to upgrade the package after installation through WinGet. For best UX, it's best to remove the user scope from the manifest

File: manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+License: Copyright Milos Paripovic (©). All Rights Reserved
+Copyright: Copyright Milos Paripovic (©). All Rights Reserved
+CopyrightUrl: https://onecommander.com/
+ShortDescription: OneCommander is a modern file manager for Windows 10 and Windows 11.
+Description: OneCommander is a modern file manager for Windows 10 and Windows 11. Features include tabs, dual-pane browsers, columns navigation, built-in preview, a theming system, color tags, and much more. It is free for home use and has no ads.
+Moniker: OneCommander
+Tags:
+- OneCommander
+- Modern
+- File-Manager
```
Comment (mdanish-kh) [inline manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml:22]:
```suggestion
Moniker: onecommander
Tags:
- onecommander
- modern
- file-manager
```
Best practice is to write moniker and tags in lower-case

File: manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MilosParipovic.OneCommander
+PackageVersion: 3.44.1
+PackageLocale: en-US
+Publisher: Milos Paripovic
+PublisherUrl: https://onecommander.com/
+PublisherSupportUrl: https://twitter.com/OneCommander
```
Comment (mdanish-kh) [inline manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml:9]:
```suggestion
PublisherSupportUrl: https://onecommander.com/help
```
A social media account is not considered as the

File: manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+PackageVersion: 3.44.1
+PackageLocale: en-US
+Publisher: Milos Paripovic
+PublisherUrl: https://onecommander.com/
+PublisherSupportUrl: https://twitter.com/OneCommander
+Author: Milos Paripovic
+PackageName: OneCommander
+PackageUrl: https://onecommander.com/
+License: Copyright Milos Paripovic (©). All Rights Reserved
+Copyright: Copyright Milos Paripovic (©). All Rights Reserved
```
Comment (mdanish-kh) [inline manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.locale.en-US.yaml:14]:
```suggestion
Copyright: Copyright Milos Paripovic © 2013-2023. All Rights Reserved.
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by BrandonWanHuanSheng** [comment]
> `user` scope for MSI installers generally does not work well because of a bug with WiX installers in that the installer installs per-user but incorrectly writes the uninstall entries to HKEY_LOCAL_MACHINE that makes WinGet think it's a per-machine install. This causes users not being able to upgrade the package after installation through WinGet. For best UX, it's best to remove the user scope from the manifest

What is the bug of MSI installer?

**Comment by mdanish-kh** [comment]
I'm not really sure if it's a bug or a design issue, but related conversation can be found here https://github.com/microsoft/winget-cli/issues/3011

**Comment by BrandonWanHuanSheng** [comment]
> I'm not really sure if it's a bug or a design issue, but related conversation can be found here [microsoft/winget-cli#3011](https://github.com/microsoft/winget-cli/issues/3011)

Thanks for reporting to the winget-cli.

**Comment by BrandonWanHuanSheng** [comment]
This error is quite happening. It look like we were unable to merged, it look like it still failed on general stage.

File: manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  InstallerType: msi
+  InstallerUrl: https://onecommander.com/OneCommanderSetup3.44.1.0.msi
+  InstallerSha256: F678326A8B7739441DABA553EE3FB8EB81477986E2D8D38CF274EFDCDBBA72CF
+  ProductCode: '{524E8270-31DB-4D4A-A420-266E7F79E86E}'
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://onecommander.com/OneCommanderSetup3.44.1.0.msi
+  InstallerSha256: F678326A8B7739441DABA553EE3FB8EB81477986E2D8D38CF274EFDCDBBA72CF
+  ProductCode: '{524E8270-31DB-4D4A-A420-266E7F79E86E}'
```
Comment (Trenly) [inline manifests/m/MilosParipovic/OneCommander/3.44.1/MilosParipovic.OneCommander.installer.yaml:21]:
```suggestion
```
> MSI (s) (A0:AC) [09:17:14:208]: Windows Installer installed the product. Product Name: OneCommander. Product Version: 3.44.1. Product Language: 1033. Manufacturer: Milos Paripovic. Installation success or error status: 1633.
> 
> This installation package is not supported by this processor type. Contact your product vendor.

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**